src/stacks-hierarchy/Root_PurchaseConfirmationScreen/Root_PurchaseConfirmationScreen.tsx

```diff
@@ -246,30 +246,10 @@ export const Root_PurchaseConfirmationScreen: React.FC<Props> = ({
   const SummaryText = () => {
     switch (zoneSelectionMode) {
       case 'multiple-stop-harbor':
-        return (
-          <>
-            {summary(
-              fareProductTypeConfig.type.includes('single')
-                ? t(
-                    PurchaseConfirmationTexts.validityTexts.harbor.single(
-                      fromPlaceName,
-                      toPlaceName,
-                    ),
-                  )
-                : t(
-                    PurchaseConfirmationTexts.validityTexts.harbor.period(
-                      fromPlaceName,
-                      toPlaceName,
-                    ),
-                  ),
-            )}
-            {summary(
-              t(
-                PurchaseConfirmationTexts.validityTexts.harbor
-                  .messageInHarborZones,
-              ),
-            )}
-          </>
+        return summary(
+          t(
+            PurchaseConfirmationTexts.validityTexts.harbor.messageInHarborZones,
+          ),
```

Comment:
This message (Which says "Applies for bus/tram in departure and destination zones") can probably be a global message. But that's outside the scope of this PR.